### ISSUE TITLE
(Re)Surface the mlbox

### ISSUE THREAD

**Issue by castrojo** [ublue-os/bluefin#1014]
This got lost in the shuffle while reorging the just files and toolboxes: 

https://github.com/ublue-os/bluefin/blob/main/dx/usr/share/ublue-os/distrobox/pytorch-nvidia.ini

Maybe we should org this, vanilla pytorch and tensorflow in one just snippet with a selector?

**Comment by nidhoggr-nil** [comment]
Personally I use something like this for ml right now, since the pytorch image is huuge and sometimes I just want cuda

```
[mlbox-pytorch]
image=nvcr.io/nvidia/pytorch:24.03-py3
additional_packages="git nvtop btop"
init_hooks="pip3 install huggingface_hub tokenizers sentencepience transformers accelerate datasets peft bitsandbytes"
#pre-init-hooks="/init_script.sh"
nvidia=true
pull=true
root=false

[mlbox-cuda]
image=nvidia/cuda:12.4.1-devel-ubuntu22.04
additional_packages="git nvtop btop"
init_hooks="pip3 install huggingface_hub tokenizers sentencepience transformers accelerate datasets peft bitsandbytes"
#pre-init-hooks="/init_script.sh"
nvidia=true
pull=true
root=false
```

**Comment by FelipeCabelloE** [comment]
where should those lines go?

**Comment by nidhoggr-nil** [comment]
I'm using this now, got tired of writing anything else than mlbox, and added some typical packages I use.

```
[mlbox]
image=nvcr.io/nvidia/pytorch:24.03-py3
additional_packages="git git-lfs"
additional_packages="zsh zsh-antigen fzf findutils unzip strace ripgrep"
additional_packages="net-tools nmap openssl rclone rsync rclone"
additional_packages="make cmake clang clang-tools gcc jq"
additional_packages="nvtop neovim tmux htop iftop iotop"
init_hooks="pip3 install huggingface_hub tokenizers sentencepiece transformers accelerate datasets peft bitsandbytes poetry"
nvidia=true
pull=true
root=false

[mlbox-cuda]
image=nvidia/cuda:12.4.1-devel-ubuntu22.04
nvidia=true
pull=true
root=false
init=true
```

It should be added in
/etc/distrobox/distrobox.ini

**Comment by karypid** [comment]
I just managed to get a ROCm container working. If I may suggest adding this flavor as well:

```
[mlbox-rocm]
image=rocm/dev-ubuntu-22.04
additional_packages="git"
pre_init_hooks="mkdir /piptmp && TMPDIR=/piptmp pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/rocm6.0"
init_hooks="pip3 install matplotlib huggingface_hub tokenizers transformers accelerate datasets peft bitsandbytes"
nvidia=true
pull=false
root=false
```
